src/test/shell/integration/execution_phase_tests.sh

```diff
@@ -64,7 +64,7 @@ function set_up() {
 }
 
 function tear_down() {
-    bazel shutdown
+  try_with_timeout bazel shutdown
```

Comment:
What is the timeout? Is it enough for `bazel shutdown`?

Reply:
120 seconds. This is defined in testenv.sh. It has been enough for me so far.

Reply:
Got it, thanks!